src/CRUD/delete.ts

```diff
@@ -15,7 +15,7 @@ export default async function deleteItem({
   key,
   options = {},
   verbose = false,
-  forTrx = false,
+  forTrx = false
```

Comment:
I do think we probably need to introduce a Prettier code-style rule file in this repository, especially given this is an open-source repository.

Reply:
There is a prettier config file in the root. The dot was removed after I ran `npm run format`. Feel free to add more rules to the existing `.prettierrc`